Compiling failed on macOS

**Describe the bug**

Hi,
I'm following the readme, download Filament's latest binary release `filament-20190423-mac.tgz` and extract into a directory. Then I set up main.cpp and makefile, but I got an error during compiling, it says `Undefined symbols for architecture x86_64:
  "_CVBufferRelease"`. Is there some dependencies I missed?

**Expected behavior**
```
$ make
$ ./main
FEngine (64 bits) created at 0x106471000 (threading is enabled)
```

**Log**

<details>

```
➜  filament make
clang++ -Iinclude/ -std=c++14 -c main.cpp
clang++ -Llib/x86_64/ main.o -lfilament -lbackend -lbluegl -lbluevk -lfilabridge -lfilaflat -lutils -lgeometry -lsmol-v -framework Cocoa -framework Metal -o main
Undefined symbols for architecture x86_64:
  "_CVBufferRelease", referenced from:
      filament::backend::metal::MetalExternalImage::set(__CVBuffer*) in libbackend.a(MetalExternalImage.mm.o)
      ____ZN8filament7backend5metal18MetalExternalImage3setEP10__CVBuffer_block_invoke in libbackend.a(MetalExternalImage.mm.o)
      std::__1::__function::__func<filament::backend::metal::$_0, std::__1::allocator<filament::backend::metal::$_0>, void (void const*)>::operator()(void const*&&) in libbackend.a(MetalExternalImage.mm.o)
  "_CVBufferRetain", referenced from:
      filament::backend::metal::MetalExternalImage::getMetalTextureForDraw() const in libbackend.a(MetalExternalImage.mm.o)
  "_CVMetalTextureCacheCreate", referenced from:
      filament::backend::metal::MetalDriver::MetalDriver(filament::backend::MetalPlatform*) in libbackend.a(MetalDriver.mm.o)
  "_CVMetalTextureCacheCreateTextureFromImage", referenced from:
      filament::backend::metal::MetalExternalImage::set(__CVBuffer*) in libbackend.a(MetalExternalImage.mm.o)
      filament::backend::metal::MetalExternalImage::createTextureFromImage(__CVBuffer*, MTLPixelFormat, unsigned long) in libbackend.a(MetalExternalImage.mm.o)
  "_CVMetalTextureCacheFlush", referenced from:
      filament::backend::ConcreteDispatcher<filament::backend::metal::MetalDriver>::endFrame(filament::backend::Driver&, filament::backend::CommandBase*, long*) in libbackend.a(MetalDriver.mm.o)
      filament::backend::metal::MetalDriver::endFrame(unsigned int) in libbackend.a(MetalDriver.mm.o)
  "_CVMetalTextureGetTexture", referenced from:
      filament::backend::metal::MetalExternalImage::set(__CVBuffer*) in libbackend.a(MetalExternalImage.mm.o)
      filament::backend::metal::MetalExternalImage::getMetalTextureForDraw() const in libbackend.a(MetalExternalImage.mm.o)
  "_CVPixelBufferGetHeightOfPlane", referenced from:
      filament::backend::metal::MetalExternalImage::set(__CVBuffer*) in libbackend.a(MetalExternalImage.mm.o)
      filament::backend::metal::MetalExternalImage::createTextureFromImage(__CVBuffer*, MTLPixelFormat, unsigned long) in libbackend.a(MetalExternalImage.mm.o)
  "_CVPixelBufferGetPixelFormatType", referenced from:
      filament::backend::metal::MetalExternalImage::set(__CVBuffer*) in libbackend.a(MetalExternalImage.mm.o)
  "_CVPixelBufferGetPlaneCount", referenced from:
      filament::backend::metal::MetalExternalImage::set(__CVBuffer*) in libbackend.a(MetalExternalImage.mm.o)
  "_CVPixelBufferGetWidthOfPlane", referenced from:
      filament::backend::metal::MetalExternalImage::set(__CVBuffer*) in libbackend.a(MetalExternalImage.mm.o)
      filament::backend::metal::MetalExternalImage::createTextureFromImage(__CVBuffer*, MTLPixelFormat, unsigned long) in libbackend.a(MetalExternalImage.mm.o)
  "_CVPixelBufferRelease", referenced from:
      filament::backend::metal::MetalDriver::cancelExternalImage(void*) in libbackend.a(MetalDriver.mm.o)
      filament::backend::metal::MetalExternalImage::set(__CVBuffer*) in libbackend.a(MetalExternalImage.mm.o)
      ____ZN8filament7backend5metal18MetalExternalImage3setEP10__CVBuffer_block_invoke in libbackend.a(MetalExternalImage.mm.o)
  "_CVPixelBufferRetain", referenced from:
      filament::backend::metal::MetalDriver::setupExternalImage(void*) in libbackend.a(MetalDriver.mm.o)
      filament::backend::metal::MetalExternalImage::getMetalTextureForDraw() const in libbackend.a(MetalExternalImage.mm.o)
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [main] Error 1

```
</details>

**Desktop (please complete the following information):**
 - OS: macOS
 - GPU: Intel Iris Plus Graphics 640 1536 MB
 - Backend: [OpenGL/Vulkan]

--- a/filament/README.md
+++ b/filament/README.md
@@ -102,7 +102,7 @@ clean:
 
 ```
 FILAMENT_LIBS=-lfilament -lbackend -lbluegl -lbluevk -lfilabridge -lfilaflat -lutils -lgeometry -lsmol-v
-FRAMEWORKS=-framework Cocoa -framework Metal
+FRAMEWORKS=-framework Cocoa -framework Metal -framework CoreVideo
 CC=clang++
 
 main: main.o

